languages/csharp/exercises/concept/chars/.docs/after.md

```diff
@@ -0,0 +1,78 @@
+- [Chars documentation][chars-docs]: reference documentation for `char`.
+- [Chars tutorial][chars-tutorial]: basic tutorial on how to work with chars.
+
+`char`s are generally easy to use. They can be extracted from strings, added back
+(by means of a string builder), initialised using literals, assigned and compared.
+However, there are a number of rough edges as detailed below.
+
+These rough edges mostly relate to the opposition
+between the full unicode standard on the one side and historic representations
+of text as well as performance and memory usage on the other.
+
+### Unicode Issues
+
+When dealing with strings, if `System.String` library methods are available you should
+seek these out and use them rather than breaking the string down into characters.
+Some textual "characters" consist of more than one `char` because the unicode standard
+has more than 65536 code points. For instance the emojis that show up in some
+of the tests have 2 `char`s as they comprise [surrogate][surrogates] characters.
+Additionally, there are combining sequences for instance where in some cases
+an accented character may consist of one `char` for the plain character
+and another `char` for the accent.
+
+If you have to deal with individual characters you should try to use
+library methods such as `System.Char.IsControl`, `System.Char.IsDigit`
```

Comment:
Feel free to link to the documentation pages of these methods.

Reply:
Done